docs/contributors.rst

```diff
@@ -6,8 +6,26 @@ Contributors
 
 All contributors to the libCellML software project agree to the terms and conditions of the `Apache v2.0 <https://opensource.org/licenses/Apache-2.0>`_ license.
 
-List of Contributors
-====================
+List of Funding Organisations
+=============================
 
  * `University of Auckland <https://www.auckland.ac.nz/>`_
  * `VPR <http://www.virtualrat.org/>`_
+
+List of Contributors
+====================
+
+The following is a list of contributors(in surname alphabetical order) who have contributed lines of
+source code to the libCellML project on or before 2019-03-05.
```

Comment:
"contributors (in" and not "contributors(in".

Also, didn't we agree that we should have one line per sentence? This makes it *much* easier to compare versions.

Reply:
Sorry that was a change because my Python code went over the PEP8 line length guideline.